### ISSUE TITLE
SimpleSQLite 0.18.0 did not install pytablereader dependency

### ISSUE THREAD

**Issue by fschim** [thombashi/SimpleSQLite#62]
When installing SimpleSQLite 0.18.0 (required by tcconfig), it did not install pytablereader:
```
Collecting SimpleSQLite>=0.17.0 (from tcconfig)
  Downloading SimpleSQLite-0.18.0-py2.py3-none-any.whl
Requirement already satisfied (use --upgrade to upgrade): setuptools>=20.2.2 in /usr/lib/python2.7/site-packages (from tcconfig)
Collecting pyparsing>=2.2.0 (from tcconfig)
  Downloading pyparsing-2.2.0-py2.py3-none-any.whl (56kB)
    100% |████████████████████████████████| 61kB 8.3MB/s
Collecting typepy>=0.0.20 (from tcconfig)
  Downloading typepy-0.0.20-py2.py3-none-any.whl
Collecting subprocrunner>=0.9.0 (from tcconfig)
  Downloading subprocrunner-0.10.0-py2.py3-none-any.whl
Requirement already satisfied (use --upgrade to upgrade): ipaddress; python_version < "3.3" in /usr/lib/python2.7/site-packages (from tcconfig)
Collecting DataProperty>=0.26.1 (from tcconfig)
  Downloading DataProperty-0.28.0-py2.py3-none-any.whl
Collecting voluptuous (from tcconfig)
  Downloading voluptuous-0.10.5.tar.gz (41kB)
    100% |████████████████████████████████| 51kB 8.5MB/s
Requirement already satisfied (use --upgrade to upgrade): six in /usr/lib/python2.7/site-packages (from tcconfig)
Collecting logbook (from tcconfig)
  Downloading Logbook-1.1.0.tar.gz (186kB)
    100% |████████████████████████████████| 194kB 5.3MB/s
Collecting pathvalidate>=0.16.2 (from SimpleSQLite>=0.17.0->tcconfig)
  Downloading pathvalidate-0.16.2-py2.py3-none-any.whl
Collecting mbstrdecoder (from SimpleSQLite>=0.17.0->tcconfig)
  Downloading mbstrdecoder-0.2.2-py2.py3-none-any.whl
Collecting tabledata>=0.0.2 (from SimpleSQLite>=0.17.0->tcconfig)
  Downloading tabledata-0.0.2-py2.py3-none-any.whl
Collecting enum34; python_version < "3.4" (from typepy>=0.0.20->tcconfig)
  Downloading enum34-1.1.6-py2-none-any.whl
Requirement already satisfied (use --upgrade to upgrade): pytz>=2017.2 in /usr/lib/python2.7/site-packages (from typepy>=0.0.20->tcconfig)
Requirement already satisfied (use --upgrade to upgrade): python-dateutil>=2.6.1 in /usr/lib/python2.7/site-packages (from typepy>=0.0.20->tcconfig)
Requirement already satisfied (use --upgrade to upgrade): futures; python_version < "3.2" in /usr/lib/python2.7/site-packages (from DataProperty>=0.26.1->tcconfig)
Installing collected packages: logbook, mbstrdecoder, enum34, typepy, DataProperty, pathvalidate, tabledata, SimpleSQLite, pyparsing, subprocrunner, voluptuous, tcconfig
  Running setup.py install for logbook ... done
  Running setup.py install for voluptuous ... done
Successfully installed DataProperty-0.28.0 SimpleSQLite-0.18.0 enum34-1.1.6 logbook-1.1.0 mbstrdecoder-0.2.2 pathvalidate-0.16.2 pyparsing-2.2.0 subprocrunner-0.10.0 tabledata-0.0.2 tcconfig-0.17.0 typepy-0.0.20 voluptuous-0.10.5
```
This caused failure in code that referenced SimpleSQLite:
```
[test@test centos]# tcset
Traceback (most recent call last):
  File "/bin/tcset", line 7, in <module>
    from tcconfig.tcset import main
  File "/usr/lib/python2.7/site-packages/tcconfig/tcset.py", line 23, in <module>
    from ._common import (
  File "/usr/lib/python2.7/site-packages/tcconfig/_common.py", line 28, in <module>
    from ._logger import logger
  File "/usr/lib/python2.7/site-packages/tcconfig/_logger.py", line 11, in <module>
    import simplesqlite
  File "/usr/lib/python2.7/site-packages/simplesqlite/__init__.py", line 23, in <module>
    from ._logger import (
  File "/usr/lib/python2.7/site-packages/simplesqlite/_logger.py", line 12, in <module>
    import pytablereader
ImportError: No module named pytablereader
```

**Comment by thombashi** [comment]
@fschim 

Thank you for your report.

Actually, I had dropped mandatory dependency for pytablereader from SimpleSQLite (at version 0.18.0).
However, the fix was incomplete.
I had released the fixed version (`SimpleSQLite 0.18.1`).
Please upgrade SimpleSQLite to workaround the issue.

I will also update the package dependency of the `tcconfig` later.

**Comment by thombashi** [comment]
I'll close the issue.
Feel free to reopen if you still have any problems.
